### PR TITLE
Add the calculator Component

### PR DESCRIPTION
**Pull request by andersonlebon**
- [ ] Delete all the boilerplate from CRA (text, images, styles).
- [ ] Create a directory called components.
- [ ] Inside components, I create a new Calculator.js file.
- [ ] Syle the component created

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+import './Sass/app.css';
 
 function App() {
```
Comment (arslanbisharat) [inline src/App.js:4]: - [ ] Please make sure you are having class based component in App.js as requested in project requested.

File: src/components/Calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  TiDivide, TiTimes, TiMinus, TiPlus, TiEquals,
+} from 'react-icons/ti';
+
+const Calculator = () => (
```
Comment (arslanbisharat) [inline src/components/Calculator.jsx:6]: - [ ] Please make sure you are having class based component in calculator.js as requested in project requested. 
- [ ] i would recommend you to not use jsx. Please use js file

File: src/tryThis.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+
+function Example() {
```
Comment (arslanbisharat) [inline src/tryThis.jsx:3]: Please remove unnecessary files from the project.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**